server/www/index.html

```diff
@@ -69,7 +69,8 @@ <h1 class="beta" ng-bind="site.name"></h1>
                             <p class="header-description" ng-bind="site.description"></p>
 
                             <div class="create-admin">
-                                <a class="button button-primary" ng-href="/posts/create">
+                              <a class="button button-primary" ng-href="/posts/create"
+                                 ng-show="!site.private || site.private && loggedin">
```

Comment:
Couldn't this just be if `loggedIn || !site.private` ? .. I'd be tempted to move this logic into a function in the controller. So you can just do `ngShow="canCreatePost()"`